crates/rolldown/src/module_finalizers/mod.rs

```diff
@@ -310,7 +310,7 @@ impl<'me, 'ast> ScopeHoistingFinalizer<'me, 'ast> {
       )),
     );
 
-    let exports_len = self.ctx.linking_info.canonical_exports(false).count();
+    let has_exports = self.ctx.linking_info.canonical_exports(false).next().is_some();
```

Comment:
`canonical_exports` is sometimes extremely long if there exists long `barrel exports` chain, this could sometimes improve a little performance.